.github/workflows/e2e_test_ci.yml

```diff
@@ -242,6 +240,8 @@ jobs:
   test_quickstart_spiceai:
     name: "E2E Test: SpiceAI quickstart, using ${{ matrix.target.target_os }}-${{ matrix.target.target_arch }}"
     runs-on: ${{ matrix.target.runner }}
+    # run quickstart with external service dependency only on manual trigger
+    if: github.event_name == 'workflow_dispatch'
```

Comment:
We can also try `if: github.event.pull_request.head.repo.full_name == github.repository`

This will disable test for PR from fork.

Reply:
We'll still have it running on each merge, let's keep it disabled for pr's